src/concrete/ml/sklearn/rf.py

```diff
@@ -124,7 +124,7 @@ def load_dict(cls, metadata: Dict):
             obj.sklearn_model,
             numpy.zeros((len(obj.input_quantizers),))[None, ...],
             framework=obj.framework,
-            output_n_bits=obj.n_bits,
+            output_n_bits=obj.n_bits["op_leaves"] if isinstance(obj.n_bits, Dict) else obj.n_bits,
```

Comment:
ideally, https://github.com/zama-ai/concrete-ml-internal/issues/4234 would avoid this kind of things, but we'll see that later I think

Reply:
You will probably encounter, this error if you define a dict in the init:

`RuntimeError: Cannot clone object DecisionTreeClassifier(n_bits=3), as the constructor either does not set or modifies parameter n_bits`

Reply:
I'm not sure, iirc we implemented `get_params` and `get_sklearn_params` to fix this specific error 🤔  but we'll see

Reply:
actually yes you're right , I've updated the issue in consequence !